design/FY2019/Refactor and cache two psychrometric functions.md

```diff
@@ -0,0 +1,77 @@
+
+**Table 1. Regression diffs of two psychrometric functions - annual**
+
+| ﻿Trade off flags       | Number of tests have large math diffs | Number of tests have large table diffs | Annual Site EUI   |                                | Annual Energy Meters Entire Facility - Electricity |                                | Annual Energy Meters Entire Facility - Gas |                                | Annual Energy Meters Entire Facility - Cooling |                                | Annual Energy Meters Entire Facility - Water |                                | Annual Energy Meters Entire Facility - Other |                                |
+|-----------------------|---------------------------------------|----------------------------------------|-------------------|--------------------------------|----------------------------------------------------|--------------------------------|--------------------------------------------|--------------------------------|------------------------------------------------|--------------------------------|----------------------------------------------|--------------------------------|----------------------------------------------|--------------------------------|
+|                       |                                       |                                        | Max relative diff | # test cases have diff > 0.01% | Max relative diff                                  | # test cases have diff > 0.01% | Max relative diff                          | # test cases have diff > 0.01% | Max relative diff                              | # test cases have diff > 0.01% | Max relative diff                            | # test cases have diff > 0.01% | Max relative diff                            | # test cases have diff > 0.01% |
+| EP_cache_PsyTsatFnPb  | 43                                    | 23                                     | 0.10%             | 7                              | 0.11%                                              | 7                              | 0.02%                                      | 2                              | 0.02%                                          | 3                              | 0.23%                                        | 12                             | 0.09%                                        | 6                              |
+| EP_cache_PsyTsatFnHPb | 57                                    | 34                                     | 0.06%             | 10                             | 0.06%                                              | 9                              | 0.02%                                      | 3                              | 0.02%                                          | 5                              | 0.18%                                        | 12                             | 0.09%                                        | 8                              |
```

Comment:
You are showing there are 7 files with an annual site EUI diff > 0.01% with EP_cache_PsyTsatFnPb.  Do you list out those files anywhere here?  I'm inclined to run tests locally to see more of what is going on.

Reply:
@Myoldmopar 
  , IDF_NAME, REL_DIFF
1, 2ZoneDataCenterHVAC_wEconomizer, 0.0010953442270432037
2, CentralChillerHeaterSystem_Cooling_Heating, 0.00033061825613885006
3, CentralChillerHeaterSystem_Simultaneous_Cooling_Heating, 0.00012790177143959987
4, CompSetPtControl, 0.0005659891947516606
5, HeatRecoveryElectricChiller, 0.00010904530832566509
6, VSHeatPumpWaterHeater, 0.00011429877700319868
7, MultiSpeedHP_StagedThermostat, 0.0001021241830065882